### PR TITLE
Added test for set and remove Objects from localStorage.

### PR DESCRIPTION
**Pull request by ManrajGrover**
Fixes indentions and typos. Added test for set and remove keys from Objects.

### CODE REVIEW

**Comment by spectralsun** [comment]
:+1:

File: shared/test/local_storage.js [hunk 1]
```diff
@@ -36,6 +36,16 @@ describe ("Local Storage Test Suite", function() {
...
-  });
+ });
 
+ it("can set and remove keys from Objects", function() {
+    var obj = {"foo": "bar", "bar": "bazz"};
+    ls.setItem("key", obj);
+    var returned = ls.getItem("key");
+    expect(ls.getItem("key")).toEqual(obj);
+    expect(returned["foo"]).toBe("bar");
+    ls.removeItem("foo");
```
Comment (smcgregor) [inline shared/test/local_storage.js]: You are never setting `foo` in localStorage so you are not removing anything here.

Comment (ManrajGrover) [inline shared/test/local_storage.js]: Sorry. What I meant is "set and remove Objects". I was trying something else back then. Fixed it. Please review the code.

**Comment by irdan** [comment]
Looks like the indentation still needs work.  `it` should be intended two spaces from the depth of `describe`, right now `it` appears to be indented by a single space.

**Comment by ManrajGrover** [comment]
@irdan sir,

Hi! :)
Major part of the file had one space in front of `it`. That's why I followed the same. If two space indention is required please let me know, will commit the change.

**Comment by smcgregor** [comment]
There are a few inconsistencies in the file's whitespace. Your change makes it more consistent within the file, but it is still contrary to our style guide and good style in general. Blocks should be indented two spaces for every level they are nested in.

**Comment by ManrajGrover** [comment]
@smcgregor sir,

Hi! :)

I have fixed the indentation for the whole file. Please review the code.

**Comment by smcgregor** [comment]
[Running on TravisCI](https://travis-ci.org/smcgregor/privly-applications/builds/59075504)

**Comment by smcgregor** [comment]
Passes. Can you squash the commits and I will merge?

**Comment by ManrajGrover** [comment]
Hi @smcgregor sir,

Sir I have squashed last two commits. Please let me know any other changes.

**Comment by smcgregor** [comment]
I think you can squash all your commits on this pull request into a single commit.

**Comment by ManrajGrover** [comment]
Hi @smcgregor sir,

I have squashed all commits. :)

**Pull request merged**